### PR TITLE
delete specific backup

### PR DESCRIPTION
**Pull request by andrianboscanean**

### CODE REVIEW

File: commands/backup.go [hunk 1]
```diff
@@ -113,3 +117,53 @@ func backupListCommand() *cli.Command {
...
+		Name:        "delete",
+		Description: "Delete existing backup file(s)",
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:        "configuration",
+				Required:    true,
+				Destination: &config,
+			},
+			&cli.StringFlag{
+				Name:        "name",
```
Comment (nicumaxian) [inline commands/backup.go:134]: can we add autocomplet?

File: commands/backup.go [hunk 2]
```diff
@@ -113,3 +117,53 @@ func backupListCommand() *cli.Command {
 		},
 	}
 }
+
+func backupDeleteCommand() *cli.Command {
+	var name string
+	var config string
+	return &cli.Command{
+		Name:        "delete",
+		Description: "Delete existing backup file(s)",
```
Comment (nicumaxian) [inline commands/backup.go:126]: i see only one operation is supported at once. you can use StringArray flag or only once file per operation.

File: commands/backup.go [hunk 3]
```diff
@@ -113,3 +117,53 @@ func backupListCommand() *cli.Command {
...
+			if specificBackup == nil {
+				return fmt.Errorf("no such file - %s", name)
+			}
+
+			s := spinner.New(spinner.CharSets[9], 100*time.Millisecond)
+			s.Start()
+			fullPath := path.Join(location, specificBackup.Name())
+			err = os.Remove(fullPath)
+			if err != nil {
+				return err
```
Comment (nicumaxian) [inline commands/backup.go:160]: add spinner.error to inform user about failure

File: commands/backup.go [hunk 4]
```diff
@@ -113,3 +117,53 @@ func backupListCommand() *cli.Command {
...
+			}
+			s.Stop()
```
Comment (nicumaxian) [inline commands/backup.go:162]: s.success

**Review by nicumaxian — CHANGES_REQUESTED** [review]

File: commands/backup.go (hunk 1 above)
Comment (andrianboscanean) [reply commands/backup.go:134]: we'll think a way,

**Review by nicumaxian — APPROVED** [review]

**Pull request merged**